File: 01.ProgrammingFundamentals/Java/01.IntroAndBasicSyntax/Exercises/src/P04_BeverageLabels.java
import java.text.DecimalFormat;
import java.util.Scanner;
public class P04_BeverageLabels {
    public static void main(String[] args) {

        Scanner console = new Scanner(System.in);
        String name = console.nextLine();
        int volume = Integer.parseInt(console.nextLine());
        int energy = Integer.parseInt(console.nextLine());
        int sugar = Integer.parseInt(console.nextLine());
        double kcal = volume*(energy/100.0);
        double g = volume*(sugar/100.0);

        DecimalFormat formatter = new DecimalFormat("#.##");
        System.out.format("%sml %s:%n",formatter.format(volume), name);
        System.out.format("%skcal, %sg sugars", formatter.format(kcal), formatter.format(g));
    }
}

File: ProgrammingFundamentals/C#/01.IntroAndBasicSyntax/Exercises/P04_BeverageLabels/P04_BeverageLabels.cs
using System;

namespace P04_BeverageLabels
{
    class P04_BeverageLabels
    {
        static void Main(string[] args)
        {
            string name = Console.ReadLine();
            int volume = int.Parse(Console.ReadLine());
            int energy = int.Parse(Console.ReadLine());
            int sugar = int.Parse(Console.ReadLine());
            double kcal = volume * (energy / 100.0);
            double g = volume * (sugar / 100.0);
            
            Console.WriteLine("{0}ml {1}:", volume, name);
            Console.WriteLine("{0}kcal, {1}g sugars", kcal, g);
        }
    }
}
